src/models/Item.js

```diff
@@ -13,11 +13,13 @@ class Item extends Model {
         item_name: { type: 'string', minLength: 1, maxLength: 255 },
         item_details: {
           type: 'object',
-          properties: {}
+          properties: {
+            description: { type: 'string' }
+          }
         }
       }
-    };
```

Comment:
The curly braces are getting all mismatched after rebasing onto master.

Reply:
should be all good now